addons/crm/static/src/scss/crm.scss

```diff
@@ -0,0 +1,35 @@
+.o_opportunity_kanban {
+
+    .oe_kanban_card {
```

Comment:
Part of css from this file can be achieved using standard bootstrap classes normally (d-flex flex-column). Only styling is custom.